vcd/resource_vcd_vapp_vm_dhcp_wait_test.go

```diff
@@ -159,6 +162,7 @@ resource "vcd_vapp_vm" "{{.VMName}}" {
   network {
     type               = "none"
     ip_allocation_mode = "NONE"
+    connected            = "false"
```

Comment:
Nit, maybe it's just GitHub, but indent shows to be off here and in all the places below in this PR:

<img width="464" alt="Screenshot 2020-07-15 at 15 14 53" src="https://user-images.githubusercontent.com/11330689/87543782-f5944c00-c6ad-11ea-9733-204f1ef9ffc1.png">

Reply:
rearanged